shared_python/TagValidator.py

```diff
@@ -54,33 +54,26 @@ def identify_category(self, tag):
 
     ## Print statements
     def print_tag_correction(self, before, after):
-        print('\r\033[96m', end='')
         self.log.info('Correction successful. "' + before + '" is now "' + after+ '"')
-        print('\x1b[0m', end='')
 
     def print_tag_warning(self, tag, tag_type, isType):
-        print('\r\033[93m', end='')
+        print()
```

Comment:
The reason you need the newline at all is because `print_progress` deliberately doesn't add a newline in order to increment the numbers in place (in this case, the `X/Y tags` number). However, you don't actually want a newline _every_ time you call `print_tag_warning`: you only need it at the point where you first realise you're going to have to interrupt the counting because there's a problem with a tag - that point would actually be just after line 186 in `validate_and_fix_tag`. I suggest you move this `print` statement there and that will hopefully fix the extra line break problem.

Reply:
Worked like a charm! Thank you! I didn't see this before! :' DDD